src/about-firmware-options.html

```diff
@@ -30,28 +30,36 @@ <h1>Currently available firmware for SkyConnect</h1>
-                        <li>This firmware is currently experimental. For more information, refer to the documentation <a href="/about-multiprotocol/" target="_blank">About
-                            multiprotocol support</a></li>
-                        <li>Other terms you may have seen to designate this firmware: multi-PAN (PAN: personal area network), RCP multi-PAN</li>
+                        <li>This is the firmware that enables SkyConnect to run both a Thread and a Zigbee network. Both networks run on the same   <abbr title="radio frequency">RF</abbr> channel.</li>
+                        <li>Bundled and automatically installed with the <b>Silicon Labs Multiprotocol</b> add-on. This add-on comes with the Thread border router component, as well as with the Zigbee stack. The ZHA integration communicates with the Zigbee stack running inside the add-on.</li>
+                        <li>This firmware is currently experimental. For more information, refer to the documentation <a href="/about-multiprotocol/" target="_blank">about
+                            multiprotocol issues</a>.</li>
+                        <li>Other terms you may have seen to designate this firmware: multi-PAN (PAN: personal area network), RCP multi-PAN.</li>
                     </ul>
-                </li>
-            </ol>
+            </p>
+            <h3>Overview of Zigbee and Thread firmware options, add-ons, and integrations in Home Assistant.</h3>
             <p class="img">
                 <picture>
                   <img src="/images/zigbee-thread-firmware-add-ons-integrations_02.png">
                 </picture>
                 <br>
-                <small>Overview of Zigbee and Thread firmware options, add-ons, and integrations in Home Assistant.</small>
-              </p></p>
+                <small>Chart illustrating the relationships between Zigbee and Thread firmware options, add-ons, and integrations in Home Assistant.</small>
+              </p>
+            </p>
+
+            <h1>Why is it best to use only one protocol at once?</h1>
+            <p>When we launched Home Assistant SkyConnect, we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio.</p>
+            <p>This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmwares for Home Assistant SkyConnect deliver the best experience to users.
```

Comment:
```suggestion
            <p>This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmware for Home Assistant SkyConnect delivers the best user experience.
```